### PR TITLE
Space age approaches

### PR DESCRIPTION
**Pull request by bobahop**
Approaches for Space Age

### CODE REVIEW

**Comment by bobahop** [comment]
@ErikSchierboom: Scala Space Age approaches ready for review.

File: exercises/practice/space-age/.approaches/applydynamic/content.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "Saturn" -> 29.447498,
+    "Uranus" -> 84.016846,
+    "Neptune" -> 164.79132
+  )
+
+  private def calculate(planet: String, seconds: Double): Double =
+    seconds / EarthSeconds / PlanetRatio.getOrElse(planet, 1.0)
+
+  def applyDynamic(methodCall: String)(seconds: Double): Double =
+    calculate(methodCall.substring(2), seconds)
```
Comment (ErikSchierboom) [inline exercises/practice/space-age/.approaches/applydynamic/content.md:24]: Oh wow, that's almost Ruby-like!

**Review by ErikSchierboom — APPROVED** [review]
Nice

**Comment by mrybak834** [comment]
The solutions here are incredible.

**Pull request merged**